tests/conftest.py

```diff
@@ -232,3 +234,10 @@ def fake_chown(path, uid, gid, **kwargs):
     mocker.patch.object(os, "chown", side_effect=fake_chown)
 
     return calls
+
+
+@pytest.fixture()
+def _meson():
+    subprocess.run(["pip", "install", "meson"], check=True)
```

Comment:
should probably do that on the ci workflow instead

Reply:
You mean install `meson` in the CI?

Reply:
yeap. We already install [a lot of](https://github.com/canonical/craft-parts/blob/main/.github/workflows/tests.yaml#L85) build tools needed by the tests